docs/conformance.md

```diff
@@ -18,6 +18,10 @@ VERSION=v0.33.0
 sonobuoy run --plugin https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/conformance.yaml
 ```
 
+To run without outside connectivity tests add the argument:
```

Comment:
@enp0s3 @oshoval  This will work only once the changes will merge and get into a release. Then we need to update the example on line 17 with the correct release number that includes the changes.
now it will not skip anything, and just run the tests when using `sonobuoy run`.
The user can also change the command to take the config yaml from he's/she's environment (located at _out/manifests/release/conformance.yaml) and it will take the latest image from the registry and it will work.

I thought it is not so bad To keep it for now as it is 
, but I can either drop this change, and add it only when the changes will get merged, or I can add note about it. 
What do you think is preferred?

Reply:
as we discussed with f2f, better to drop stuff that is not yet supported and add it again when it does please

Reply:
Done.